modules/veteran/lib/tasks/veteran_tasks.rake

```diff
@@ -16,13 +16,15 @@ namespace :veteran do
     Veteran::Service::Organization.create(poa: '095', name: '095 - ITALIAN AMERICAN WAR VETERANS OF THE US, INC.')
     Veteran::Service::Organization.create(poa: '1NY', name: '1NY - SAMANTHA Y WARSHAUER')
     Veteran::Service::Representative.create(
+      representative_id: '98765',
```

Comment:
FWIW, when @SlakrHakr and I were working on the cleaning up the existing POA data so that it would conform to the new constraints, we set Tamara's `representative_id` to `999999999999`.